.pre-commit-config.yaml

```diff
@@ -17,18 +17,18 @@ repos:
     exclude: .patch
 
 - repo: https://github.com/psf/black
```

Comment:
For a faster mypyc-compiled wheel:
```suggestion
- repo: https://github.com/psf/black-pre-commit-mirror
```

Reply:
Not for right now, but maybe at some point we should switch to ruff instead of black? Presumably it's even faster...

https://docs.astral.sh/ruff/formatter/

Reply:
:+1: I use the ruff formatter on several of my projects and have been very satisfied with it.